style.css

```diff
@@ -0,0 +1,172 @@
+@import url('https://fonts.googleapis.com/css2?family=Rock+Salt&display=swap');
+
+* {
+  margin: auto;
+  box-sizing: border-box;
+  font-family: 'Rock Salt', cursive;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  background-color: green;
+  font-family: 'Rock Salt', cursive;
+  text-align: center;
+  overflow-x: hidden;
+}
+
+header {
+  display: flex;
+  position: absolute;
+  width: 100%;
+  height: 20%;
+  color: navy;
+  background: deepskyblue url('images/hill.svg') no-repeat center bottom;
+}
+
+header span {
+  display: inline-block;
+  text-align: center;
```

Comment:
This line isn't doing anything, text-align center is already applied in the body